pkg/proxy/ipvs/proxier.go

```diff
@@ -689,15 +688,8 @@ func cleanupIptablesLeftovers(ipt utiliptables.Interface) (encounteredError bool
 		"-m", "comment", "--comment", "kubernetes service portals",
 		"-j", string(kubeServicesChain),
 	}
-	tableChainsWithJumpServices := []struct {
```

Comment:
I think the original form is easier to read.  Less LOC is not always "better"

Reply:
In fact, this same table is created twice - can we define it as a var outside of any function, with a good name?